api/tests/src/api/opportunities_v1/test_opportunity_route_get.py

```diff
@@ -88,6 +90,27 @@ def test_get_opportunity_with_attachment_200(
     validate_opportunity_with_attachments(opportunity, response_data)
 
 
+def test_get_opportunity_with_attachment_200_legacy(
+    client, api_auth_token, enable_factory_create, db_session
+):
+    # Create an opportunity with an attachment
+    opportunity = OpportunityFactory.create(has_attachments=True)
+    db_session.commit()
+
```

Comment:
I know it's done in some tests, but it's something we should cleanup - we don't need to commit after using factories, the factories auto-commit after creating things.